### PR TITLE
Feat/honeycomb harvest all

### PR DESCRIPTION
**Pull request by jiacomm**

### CODE REVIEW

File: src/components/Farming/DepositTable/index.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { getNetworkConfig } from '../../../networks'
 import UserErrorScreen from '../../Errors/UserErrorScreen'
 import { truncateDecimals } from '../../../lib/math-utils'
 import styled from 'styled-components'
+import HarvestAll from '../HavestAll'
```
Comment (Philogy) [inline src/components/Farming/DepositTable/index.js]: Spelling error

File: src/components/Farming/DepositTable/index.js [hunk 2]
```diff
@@ -101,6 +102,33 @@ const DepositTable = props => {
     setErrorVisible(false)
   }
 
+  const wrapBatchAction = items => {
+    let dollarSum = 0
+    let amountSum = 0
+    let rewardBalanceSum = 0
+    items.forEach(
```
Comment (Philogy) [inline src/components/Farming/DepositTable/index.js]: please opt to using a `.reduce()` pattern.

File: src/components/Farming/HavestAll/index.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  const [visible, setVisible] = useState(false)
+  const [txHash, setTxHash] = useState('')
+  const opener = useRef()
+  const harvestAll = useHarvestAll(props.ids, chainId)
+  const network = getNetworkConfig(chainId)
+
+  const transactionTime = new Date()
+  transactionTime.setSeconds(transactionTime.getSeconds() + 8)
+
+  const handleHarvestAll = () => {
```
Comment (Philogy) [inline src/components/Farming/HavestAll/index.js:21]: Please use async / await

File: src/components/Farming/DepositTable/index.js [hunk 4]
```diff
@@ -101,6 +102,31 @@ const DepositTable = props => {
     setErrorVisible(false)
   }
 
+  const wrapBatchAction = items => {
+    const dollarSum = items.reduce(
+      (acc, cur) => acc + Number(calculateDollar(cur.amount, cur.pairInfo)),
+      0
+    )
+    const amountSum = items.reduce((acc, cur) => acc + Number(cur.amount), 0)
+    const rewardBalanceSum = items.reduce(
```
Comment (pxbee) [inline src/components/Farming/DepositTable/index.js:111]: Please use truncateDecimals on dollar amount and sum as well

**Review by pxbee — CHANGES_REQUESTED** [review]

File: src/networks.js [hunk 5]
```diff
@@ -6,6 +6,7 @@ export const networkConfigs = {
     ReferralRewarder: '0x82374C59709AAc2f7864191a3c492932379536F4',
     StreamedAirdropper: '0xdD36008685108aFafc11F88bBc66C39A851Df843',
     xCombToken: '0x38Fb649Ad3d6BA1113Be5F57B927053E97fC5bF7',
+    multiWithdrawer: '0x53f224f83b2b2365caf4178f52c234da1ecf392f',
```
Comment (pxbee) [inline src/networks.js:9]: Please add multiwithdrawer for polygon as well. You can find the address here https://github.com/1Hive/honeyswap-farm/blob/polygon-airdrop/addresses.json

**Review by pxbee — CHANGES_REQUESTED** [review]
Please merge latest master deployment to get the correct addresses in the network.js

**Pull request merged**